test/test_other.py

```diff
@@ -6750,14 +6750,15 @@ def test_bad_locale(self):
     self.assertContained('locale set to waka: waka',
                          self.run_js('a.out.js', args=['waka']))
 
-  @with_env_modify({'LC_ALL': 'en_US'})
+  @crossplatform
```

Comment:
Did you find that setting this @ crossplatform caused the failure to reproduce on CircleCI? I ask because I couldn't get it to repro locally so I suspect it's something wonky about the python that Chromium infra uses.

Reply:
Yup I think your are right.  Likely the machine didn't have that particular locale setup.